packages/terminology/src/sqlClient.ts

```diff
@@ -0,0 +1,65 @@
+import { Database } from "sqlite3";
+/* eslint-disable */
+
+export class SqliteClient {
```

Comment:
Since we're building this, I'd have an interface `DbClient` and a class `SqliteDbClient` that implements that interface, with a factory that for now just returns a new instance of the class.